Packages/com.unity.inputsystem/InputSystem/Actions/InputBinding.cs

```diff
@@ -67,6 +86,13 @@ public string name
             set => m_Name = value;
         }
 
+        /// <summary>
+        /// Unique ID of the binding.
+        /// </summary>
+        /// <value>Unique ID of the binding.</value>
+        /// <remarks>
+        /// TODO
+        /// </remarks>
         public Guid id
```

Comment:
I wonder. If we change InputDevice.id -> deviceId with the reasoning that "two letter identifiers suck", should we change this one accordingly?

Reply:
Also: TODO in public docs. Let's make sure we don't forget to replace it with content!

Reply:
Think it makes sense to change them. Maybe change all three of them (`InputAction.id`, `InputActionMap.id`, and `InputBinding.id`) to `guid`?

>Also: TODO in public docs. Let's make sure we don't forget to replace it with content!

Yup, there will be one more PR with docs. In that one, I intend to get rid of all remaining TODOs in the doc comments. ATM they are still quite helpful for me to keep track of what I only partially finished.